public/content/documentation/web/component/informative-image.md

```diff
@@ -6,75 +6,140 @@ How to test an informative image
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test an informative image
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Arrow-keys: Screen reader reads the alt text (ignored if decorative)
```

Comment:
for all of these gherkin/condensed, I think we should do a re-write to get rid of "(ignored if decorative)" as that does not apply at all here and we have separate criteria for decorative images